conf/secrets/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Can we include either a comment in this .gitignore file or a Readme in this dir with an example use case?